CONTRIBUTING.md

```diff
@@ -107,3 +107,38 @@ Icons are located in [packages/react/src/components/Icon/icons](./packages/react
 - Remove any namespace attributes (such as `xmlns:*`) from the SVG.
 - Save your icon in the above location with the proposed name of your icon, e.g. `[your-name].svg`. The `your-name` portion will be how the icon is used in the React Component, e.g. `<Icon type="your-name" />`.
 - Submit a new `feat` PR with your changes.
+
+## Breaking Changes
+
+Cauldron is constantly evolving, and it's inevitable that breaking changes will need to be introduced. Before a breaking change is released, other alternatives should be considered and allow for a deprecation period before the breaking change is released.
+
+### Components
+
+For cauldron's component library, any of the following would be considered breaking changes:
+
+* Renaming (exported name) or removing a Component
+* Changing a prop that would break usage of an existing component
+* Adding a new required prop to an existing component
+
+### Styles
+
+For cauldron's styles library, any of the following would be considered breaking changes:
+
+* Changing or removing a class name
+* Changing or removing a global variable (see `variables.css`)
+* Changing or removing a component variables
+
+In addition, changing a css property may be a breaking change for the following:
+
+* Changing a global variable property (see `variables.css`)
+* Changing a component variable property
+
+For the latter, the change must be reviewed by a member of the cauldron team to ensure there are no breaking changes.
+
+### Deprecating
+
+Before a breaking change can be released, the breaking change should be documented with the component, property, or style being changed getting flagged as _Deprecated_. This could meaning warning the consumer that a component or property is now deprecated, or including a deprecated comment next to a css class name. This deprecation must exist for at least two months.
```

Comment:
```suggestion
Before a breaking change can be released, the breaking change should be documented with the component, property, or style being changed getting flagged as _Deprecated_. This could mean warning the consumer that a component or property is now deprecated, or including a deprecated comment next to a css class name. This deprecation must exist for at least two months.
```

Reply:
Also, this section seems to make the assumption that breaking changes entail a deprecation, but sometimes the deprecation is additive (e.g. a new required prop). What is the process there?

Reply:
Good question. In this instance you would need to add the new prop as non-required and warn that in the future that property will become required. Alternatively, don't make the property required but set a default value if not set (if possible).

Reply:
> Good question. In this instance you would need to add the new prop as non-required and warn that in the future that property will become required. Alternatively, don't make the property required but set a default value if not set (if possible).

Can we document that?